packages/block-library/src/video/block.json

```diff
@@ -77,6 +77,10 @@
 	"supports": {
 		"anchor": true,
 		"align": true,
+		"spacing": {
+			"margin": true,
+			"padding": true
```

Comment:
I noticed a slight issue with padding in TwentyTwentyTwo that I think is caused by the Video block using a `width: 100%` rule, in that the padding winds up being outside the `100%` as opposed to part of it. Here's a Video block with padding next to a Group block with padding:

<img width="780" alt="image" src="https://user-images.githubusercontent.com/14988353/185520640-7d665ddf-59e0-413a-acd3-f5290a4a4fa0.png">

Shall we add a `box-sizing: border-box` rule as in the Audio block PR (It looks like most blocks with padding have the rule added, I think, along with a comment explaining why the rule was required) https://github.com/WordPress/gutenberg/pull/43351#pullrequestreview-1076698428

Reply:
Thanks for finding that. **TIL**

I'll add the rule and retest. 🙇

Reply:
Looks like that did the trick. Thank you!!!!
<img width="737" alt="Screen Shot 2022-08-19 at 12 30 28 pm" src="https://user-images.githubusercontent.com/6458278/185530036-5a4e3080-4cd7-4306-90e6-ae535faa7e67.png">